libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -346,6 +359,49 @@ public QnAMakerDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
         [JsonProperty("rankerType")]
         public StringExpression RankerType { get; set; } = new StringExpression(RankerTypes.DefaultRankerType);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to include precise answer in response. 
+        /// </summary>
+        /// <value>
+        /// True or false to include precise answer in response, defaults to false. 
+        /// </value>
+        [JsonProperty("enablePreciseAnswer")]
+        public bool EnablePreciseAnswer { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to include unstructured sources in search for answers. 
+        /// </summary>
+        /// <value>
+        /// True/False indicating whether to include unstructured sources, defaults to false. 
+        /// </value>
+        [JsonProperty("includeUnstructuredSources")]
+        public bool IncludeUnstructuredSources { get; set; }
```

Comment:
TODO: Gabo need to check if this should be BoolExpression instead of native type